modules/las/src/lib/parse-las.js

```diff
@@ -12,6 +12,7 @@ function detectTwoByteColors(colorDepth, decoder, batchSize) {
       twoByteColor = true;
       break;
     case 'auto':
+      if (!decoder.getPoint(0).color) break;
```

Comment:
Just nits (which means fix not required)

I would have thought the linter should have complained here. We usually require braces around if statement clauses and also we typically disallow extra breaks and continues. I wonder if linter is disabled here?

Wrapping this case in `if (decoder.getPoint(0).color) {` would have been more in line with the existing style.

Reply:
Noted and fixed !